### DISCUSSION TITLE
Ifc Link

### DISCUSSION THREAD

**Discussion by Alx-88** [IfcOpenShell/IfcOpenShell#3265] [General]
Hello, everyone,
a perhaps obvious question, but how come if I use the "Ifc Link" function, the elements in the linked model appear to me without parameters?
Thanks!

**Comment by Moult** [comment]
Yes, unfortunately that is the current situation with linked models - they are purely there as a geometric reference.

**Reply by Alx-88** [reply]
I see, thank you!
